src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -224,10 +227,30 @@ public Position getPosition() {
         });
   }
 
+  public CompletableFuture<List<PersonPositionHistory>> loadAllPeoplePositionHistory() {
```

Comment:
This method is only called by `getAllPeoplePositionHistory` below.

Reply:
This method is removed.